doc/input-output-reference/src/overview/group-air-distribution-equipment.tex

```diff
@@ -371,7 +371,7 @@ \subsubsection{Outputs}\label{outputs-2-000}
 
 This output is the value for the minimum air flow fraction setting on a VAV terminal.~~ This is the average value over the frequency being reported. The minimum air flow fraction is defined as the lower limit on the terminal unit air mass flow rate divided by the terminal units maximum air mass flow rate.
 
-\paragraph{Zone Air Terminal Outdoor Air Volume Flow Rate {[}m3/s{]}}
+\paragraph{Zone Air Terminal Outdoor Air Volume Flow Rate {[}m3/s{]}}\label{zone-air-terminal-outdoor-air-volume-flow-rate-m3s}
```

Comment:
You can't use the same label over and over for every individual paragraph.  You don't have to label them unless you plan on cross-referencing that field in the text.  This is why CI is complaining.  The label was first declared on line 73, and all the others are duplicates of that.

Reply:
I was think every individual air terminal unit entry needs its own output variable declaration including the label. OK, I will remove the duplicate labels.

Reply:
:+1: